src/Compilers/CSharp/Test/Semantic/Semantics/ImplicitObjectCreationTests.cs

```diff
@@ -4533,5 +4533,60 @@ static void Main()
             Assert.Equal(CandidateReason.None, symbolInfo.CandidateReason);
             Assert.Equal(0, symbolInfo.CandidateSymbols.Length);
         }
+
+        [Fact]
+        [WorkItem(50489, "https://github.com/dotnet/roslyn/issues/50489")]
+        public void InEarlyWellknownAttribute()
```

Comment:
> Do we have a test covering a bad case like this? Looking through the early attribute binding code I don't see a case that this would cause a recursion, but it would be good to add.

I do not think suggested scenario is going to exercise the modified code, C is not a Wellknown  attribute. 

---
In reply to: [566382589](https://github.com/dotnet/roslyn/pull/50862#discussion_r566382589) [](ancestors = 566382589)